canonical-web-and-design/maas-ui
# feat(ui): make the machine name form reusable

## Done

- Update the machine name for to be reusable by devices.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to the details page for a machine.
- Click on the name in the header.
- Enter a new name and domain and submit the form.
- The form should close and the machine's name and domain should update.

## Fixes

Fixes: canonical-web-and-design/app-tribe#553.
